src/query/interpret/awesome_memgraph_functions.cpp

```diff
@@ -593,6 +593,7 @@ TypedValue ValueType(const TypedValue *args, int64_t nargs, const FunctionContex
     case TypedValue::Type::Duration:
       return TypedValue("DURATION", ctx.memory);
     case TypedValue::Type::Graph:
+    case TypedValue::Type::Function:
```

Comment:
Why here could not returned empty graph and empty std::function?

Reply:
This is part of API towards functions, I think we don't need it, and if we do there is bug somewhere. Graph is only used for procedures and should be used that way, Function is only used on defering evaluation on operator, so it shouldn't yet be used here. If it will be, then someone can update it in my opinion.